data-model/virtualcollections/v.0.1.0/schemas/virtualcollections.json

```diff
@@ -1 +1,178 @@
+    "examples": [
+        true
+    ]
+              }
+            },
+            "description": "An intentionally grouped set of objects with one or more common characteristics."
+        },
+        "ltc:MeasurementOrFact": {
+            "type": "object",
+            "properties": {
+                "dwc:measurementType": {
+                    "type": "string",
+                    "examples": [
+                        "Object Count"
+                    ],
+                    "description": "https://tdwg.github.io/ltc/terms/index.html#MeasurementOrFact_measurementType The nature of the measurement, fact, characteristic, or assertion."
+                },
+                "dwc:measurementValue": {
+                    "type": "number",
+                    "examples": [
+                        100
+                    ],
+                    "description": "https://tdwg.github.io/ltc/terms/index.html#MeasurementOrFact_measurementValue The numerical value of the measurement."
+                },
+                "dwc:measurementUnit": {
+                    "type": "string",
+                    "examples": [
+                        "mm, C, km, count, percentage, category"
+                    ],
+                    "description": "https://tdwg.github.io/ltc/terms/#MeasurementOrFact_measurementUnit"
+                }
+            },
+            "description": "https://tdwg.github.io/ltc/terms/index.html#MeasurementOrFact_MeasurementOrFact A measurement of or fact about a class within the standard, or a relationship between the ObjectGroup and an associated class."
+        },
+        "ods:searchFilters": {
+            "type": "array",
+            "description": "Search filters selected for the virtual collection.",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "filterValue": {
+                        "type": "array",
+                        "examples": [
+                            "topicDiscipline=Zoology&midsLevel=1&country=Indonesia&hasMedia=1"
+                        ],
+                        "description": "Value or configuration of the filter."
+                    }
+                }
+            }
+        },
+        "ltc:basisOfScheme": {
+            "type": "string",
+            "examples": [
+                "dataset for workshop"
+            ],
+            "description": "A summary of the basis or purpose for the LatimerCoreScheme. This property is intended to summarize the reason for grouping a number of ObjectGroups within the LatimerCoreScheme, and the purpose for which the data is intended to be used."
+        }
+    },
+    "required": [
+        "rdf:type",
+        "ods:id",
+        "ods:version",
+        "ltc:ObjectGroup",
+        "ltc:collectionName"
+    ],
+    "additionalProperties": false,
+    "examples": [ 
```

Comment:
Oeh I wouldn't put this in the file. It does bloat it a bit and isn't very finable. I also noticed that it doesn't validate. 
Two issues (says https://www.jsonschemavalidator.net/)
 Found 2 error(s)
Message:
Invalid type. Expected Boolean but got Array.
Schema path:
https://schemas.dissco.tech/schemas/digitalobjects/0.1.0/virtualcollections/virtualcollections.json#/properties/ltc:ObjectGroup/properties/ltc:isKnownToContainTypes/type
Message:
Required properties are missing from object: ltc:collectionName.
Schema path:
https://schemas.dissco.tech/schemas/digitalobjects/0.1.0/virtualcollections/virtualcollections.json#/required

Reply:
removed the example. will add to the examaple folder.